docs/Running-Mastodon/Production-guide.md

```diff
@@ -34,9 +34,17 @@ server {
   keepalive_timeout    70;
   sendfile             on;
   client_max_body_size 0;
-  gzip off;
 
-  root /home/mastodon/live/public;
```

Comment:
You can't remove this line. We *want* to serve static assets through nginx, rather than making Ruby/Rails do it. Please explain your reasoning!